lib/galaxy/tools/deps/conda_util.py

```diff
@@ -309,7 +324,20 @@ def is_target_available(conda_target, conda_context=None):
 
 def is_conda_target_installed(conda_target, conda_context=None):
     conda_context = _ensure_conda_context(conda_context)
-    return conda_context.has_env(conda_target.install_environment)
+    if conda_context.has_env(conda_target.install_environment):
+        tempdir = tempfile.mkdtemp(prefix="jobdeps")
```

Comment:
Do we need a directory here? Maybe an StringIO.StringIO() object would do the trick?